scripts/core/superdesk-api.d.ts

```diff
@@ -220,7 +220,7 @@ declare module 'superdesk-api' {
         sign_off: string;
         feature_media?: any;
         media_description?: string;
-        associations?: { string: IArticle };
+        associations?: any;
```

Comment:
It should probably be `IRelatedArticle`. Even though the entire `IArticle` is stored in some cases, I think for items which aren't locally in superdesk.

Reply:
oops, forgot about that